tests/FakeItEasy.Specs/DefaultInterfaceMethodSpecs.cs

```diff
@@ -0,0 +1,89 @@
+﻿namespace FakeItEasy.Specs;
+
+using System;
+using FakeItEasy.Configuration;
+using FakeItEasy.Tests.TestHelpers;
+using FluentAssertions;
+using LambdaTale;
+using Xunit;
+
+#if !LACKS_DEFAULT_INTERFACE_METHODS
+public static class DefaultInterfaceMethodSpecs
+{
+    public interface IHaveDefaultMethods
+    {
+        int Foo() => 42;
+
+        sealed int Bar() => 123;
+    }
+
+    [Scenario]
+    public static void FakeInterfaceWithDefaultMethod(IHaveDefaultMethods fake, int result)
+    {
+        "Given a faked interface with a default method"
+            .x(() => fake = A.Fake<IHaveDefaultMethods>());
+
+        "When the default interface method is called"
+            .x(() => result = fake.Foo());
+
+        "Then it should return the default value"
+            .x(() => result.Should().Be(0));
+    }
+
+    [Scenario]
+    public static void FakeInterfaceWithDefaultMethodWithCallsBaseMethods(IHaveDefaultMethods fake, int result)
+    {
+        "Given a faked interface with a default method configured to call base methods"
+            .x(() => fake = A.Fake<IHaveDefaultMethods>(o => o.CallsBaseMethods()));
+
+        "When the default interface method is called"
+            .x(() => result = fake.Foo());
+
+        "Then it should return the value from the default implementation"
+            .x(() => result.Should().Be(42));
+    }
+
+    [Scenario]
+    public static void FakeInterfaceWithDefaultMethodOverrideBehavior(IHaveDefaultMethods fake, int result)
+    {
+        "Given a faked interface with a default method"
+            .x(() => fake = A.Fake<IHaveDefaultMethods>());
+
+        "When the sealed default method is configured to return something else"

```

Comment:
```suggestion
        "When the default method is configured to return something else"
```

or maybe a non-sealed? So long as you're not saying the method is sealed, I'm happy

Reply:
Oops. Bad copy/paste, I guess.

Reply:
Amended